met-web/src/components/engagement/view/PreviewBanner.tsx

```diff
@@ -55,15 +60,43 @@ export const PreviewBanner = () => {
             }}
         >
             <Grid container direction="row" justifyContent="flex-end" alignItems="flex-start" padding={4}>
-                <Grid item xs={12}>
-                    <Typography variant="h4">Preview Engagement{!isDraft && ' - Published'}</Typography>
+                <Grid item container xs={12}>
+                    <Typography variant="h4" sx={{ mb: 2 }}>
+                        Preview Engagement{!isDraft && ' - Published'}
+                    </Typography>
                     <ConditionalComponent condition={isDraft}>
+                        <ConditionalComponent condition={imageExists === ''}>
```

Comment:
imageExists, the name implies it is a boolean, but here you are checking if it equals a string? 

condition={!savedEngagement.banner_url} ?

Reply:
Typescript gave me a boolean error and it asked for me to change it to a string

Reply:
I took this code from somewhere else in the codebase